src/Base/Console.h

```diff
@@ -27,13 +27,11 @@
 #include <Base/PyExport.h>
 #include <Base/Stream.h>
 //#pragma warning(disable: 4786)  // specifier longer then 255 chars
-#include <assert.h>
-#include <set>
+#include <chrono>
 #include <map>
+#include <set>
 #include <string>
-#include <cstring>

```

Comment:
std::strrchr is used in Console.cpp. So it should be possible to move cstring from here to Console.cpp

Reply:
Done.